### ISSUE TITLE
[ACCESSIBILITY] Create poll text input field cut off - low-vision users may experience issues

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1493]
**Describe the accessibility issue**
When low-vision user is creating a poll, they are unable to see the text they are entering as it is being cut off by the form field itself. 

[WCAG 1.4.12 Text Spacing (AA)](https://www.w3.org/TR/WCAG21/#text-spacing)
**Info about guideline:**
In content implemented using markup languages that support the following [text]() [style properties](), no loss of content or functionality occurs by setting all of the following and by changing no other style property:
  Line height (line spacing) to at least 1.5 times the font size;
 Spacing following paragraphs to at least 2 times the font size;
 Letter spacing (tracking) to at least 0.12 times the font size;
  Word spacing to at least 0.16 times the font size.

**To Reproduce
Steps to reproduce the behavior:**

1. Go to 'Quick Polls.'
3. Click on 'Create Poll.'
4. Form for poll loads
5. User populates fields: Question, Option 1, Option 2

**Expected Behavior**
User attempts to confirm what they input in fields but words are cut off and they may not be able to perceive the info.

**Screenshots**
![Screenshot_20220323-105205](https://user-images.githubusercontent.com/82892873/160189753-d4de74e2-4cb0-4725-b692-3c4ff7e00450.png)

**Smartphone (please complete the following information):**
iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Additional context**

**Comment by acovher2** [comment]
Verified as fixed. Closing.
